models/metrics/metrics.yml

```diff
@@ -4,12 +4,12 @@ metrics:
     label: Weekly Email Delivered Events
     model: ref('hubspot__email_events')
     description: "Weekly number of email delivered events."
-    type: count
-    sql: event_id
+    calculation_method: count
+    expression: event_id
     timestamp: created_timestamp
     time_grains: [week]
     filters:
-      - field: event_type
+      - field: event_calculation_method
```

Comment:
Incorrect change, we should only be changing `type` to `calculation_method` for the metric spec identifiers

Reply:
Made changes. Please check